### ISSUE TITLE
Manually trigger completion and lazy load nvm?

### ISSUE THREAD

**Issue by tmpm697** [neoclide/coc.nvim#3065]
**Is your feature request related to a problem? Please describe.**
I lazy load nvm when I issue commands node, npm, nvim -- So only when I issue those commands, nvm will be loaded (~$NVM_DIR/nvm.sh),  this nvm.sh script took 1-2s to load and nvim will freeze for the first time startup.

For some reasons I'd like to keep using nvm, and I often open file like .sh, .js but just for viewing and no intend to have fully fledged completion avail  to use.

I want to trigger coc completion manually and then load nvm.sh before that. How to?

**Describe the solution you'd like**
I'd like an option to map to a key to trigger completion and load nvm.sh (at that time took 1-2s is acceptable).

Basically I don't want every time I start nvim, it freezes for 1-2s for 1st time (I use tmux and create/destroy new pane with nvim alot), I want nvim freezes only when I need coc.nvim and only at that time nvm.sh should be loaded.

**Describe alternatives you've considered**
I don't have.

Reference links:
https://github.com/neoclide/coc.nvim/issues/856

**Comment by tmpm697** [comment]
I installed "yarn" -- I use archlinux, it requires node installed (latest version from offical repo).

this node version comes from system and nvim seems to recognize instantly that will freeze it 1-2s if from nvm.

but I did lazy load nvm when using node/npm commands then when I use node/npm commands, they will points to nvm versions.

So this seems a workaround solution, worked for me long time but until I decided to have nvm version only.
